src/App.js

```diff
@@ -1,30 +1,45 @@
 import React from 'react';
-
-import './App.scss';
+import PostList from './components/PostList';
 
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+import './App.scss';
+
+const preparedPosts = posts.map(post => (
+  {
+    ...post,
+    user: users.find(user => post.userId === user.id),
+    comment: comments.filter(comment => post.id === comment.postId),
```

Comment:
```suggestion
    comments: comments.filter(comment => post.id === comment.postId),
```
It's an array of comments, not just single comment